src/cli/menuItems/systemCommands.cpp

```diff
@@ -15,7 +15,6 @@
 
 void CLI_connect(void)
 {
-    Particle.connect();
```

Comment:
Why did we remove this?

Reply:
Fixed, I was actually wondering why the connect command just hanged, that would explain it